scripts/generate-ozone-info.groovy

```diff
@@ -0,0 +1,9 @@
+import groovy.json.JsonOutput
+
+def json = [
+        name: 'ozone',
```

Comment:
I've hardcoded this because; The artifactId for ozone/distro is `ozone-distro`, and this will always be `ozone` except for child implementation which has an additional task to update `ozone-info` accordingly.